main.cpp

```diff
@@ -170,20 +192,29 @@ struct ElectricGuitar
     int numPickUps = 2;
     int numPickupPositions = 5;
     int numKnobs = 3;
+    ElectricGuitar(int numBrokenStrings);
 
     float amplifyStringVibration(int stringNum, float stringFreq);
     float warmTone (float noteInfo);
     float brightTone (float noteInfo);
 };
+
+ElectricGuitar::ElectricGuitar(int numBrokenStrings)
+{
+    ElectricGuitar::numStrings-=numBrokenStrings;
+}
```

Comment:
member variables do not need full-name qualification
add space around your operators `-=`  so your code is readable.
- [ ] revise